src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY_STRING = "supply";
+    private static final String BUY_STRING = "buy";
+    private static final String RESULT_STRING = "result";
+    private static final int INDEX_OF_STRING = 0;
+    private static final int INDEX_OF_VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try {
+            List<String> strings = Files.readAllLines(file.toPath());
+
+            for (String str : strings) {
+                String[] arr = str.split(",");
+                if (arr[INDEX_OF_STRING].equals(SUPPLY_STRING)) {
+                    totalSupply += Integer.valueOf(arr[INDEX_OF_VALUE]);
+                } else if (arr[INDEX_OF_STRING].equals(BUY_STRING)) {
+                    totalBuy += Integer.valueOf(arr[INDEX_OF_VALUE]);
```

Comment:
`Integer.parseInt(arr[INDEX_OF_VALUE]); `you can extract this operation from if-else and here only use extracted variable.